tests/cli-tests/cltools/zstdgrep.sh.stderr.glob

```diff
@@ -0,0 +1 @@
+zstd: can't stat bad.zst : No such file or directory -- ignored
```

Comment:
nit: This could be `zstdgrep.sh.stderr.exact`.

If you run with `./run.py --preserve`, you can `cp scratch/cltools/zstdgrep.sh/stderr cltools/zstdgrep.sh.stderr.exact`